plugin/pkg/admission/antiaffinity/admission.go

```diff
@@ -59,7 +60,8 @@ func (p *plugin) Admit(attributes admission.Attributes) (err error) {
 	}
 	affinity, err := api.GetAffinityFromPodAnnotations(pod.Annotations)
 	if err != nil {
-		return err
+		glog.V(5).Infof("Invalid NodeAffinity detected, but we will leave handling of this to validation phase")
```

Comment:
s/NodeAffinity/Affinity/g

This function is to parse Affinity not NodeAffinity.